src/app/shared/components/search-input/search-input.component.ts

```diff
@@ -26,6 +21,11 @@ export class SearchInputComponent implements OnInit, OnDestroy {
 
   public ngOnInit() {
     this.subscribeToSearch();
+    this.cardsStore.searchTerm$.pipe(
+      takeUntil(this.destroy)
+    ).subscribe((search) => {
+      this.searchText.setValue(search, { emitEvent: false });
+    });
```

Comment:
This is like the one component that needs to do something when the search changes. The input should reflect what the current search term is.